src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.tsx

```diff
@@ -147,9 +147,9 @@ export default function SideloadDialog({
         executable: selectedExe,
         platform: gameInfo.install?.platform ?? platformToInstall
       },
-      art_cover: imageUrl,
+      art_cover: imageUrl ? imageUrl : 'fallback',
```

Comment:
I think this one and the one below can be fixed by changing this line:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/560e6efd9d81df987da9e861fcca37411a22bd46/src/backend/storeManagers/sideload/library.ts#L18

to 

```
  art_cover = 'fallback',
  art_square = 'fallback',
```

So no need to change the frontend